recipes/sdl/all/conanfile.py

```diff
@@ -192,6 +196,10 @@ def _configure_cmake(self):
                 self._cmake.definitions["PULSEAUDIO"] = self.options.pulse
                 self._cmake.definitions["SNDIO"] = self.options.sndio
                 self._cmake.definitions["NAS"] = self.options.nas
+                if self.options.nas:
+                    cmake_extra_ldflags += ["-lXau"]  # FIXME: SDL sources doesn't take into account transitive dependencies
```

Comment:
Re your FIXME. Upstream `nas` does not install a `.pc` file, so there's not much sdl can do about this.

Reply:
Yes it is. Generator `pkg_config` is declared and the file is created:

```
sdl/2.0.16: Generator pkg_config created nas.pc
```

Maybe I'm missing something from your comment?

Reply:
I meant to say is that, because the upstream Nas project does not provide a pic file, SDL will never look for one.
So the fixme lies with nas and SDL.